### ISSUE TITLE
bind key errors

### ISSUE THREAD

**Issue by burtonwilliamt** [burtonwilliamt/dotfiles#2]
Getting the following error for tmux 2.7:

```
/Users/ladmin/.tmux.conf:53: usage: bind-key [-cnr] [-T key-table] key command [arguments]
/Users/ladmin/.tmux.conf:54: usage: bind-key [-cnr] [-T key-table] key command [arguments]
/Users/ladmin/.tmux.conf:55: usage: bind-key [-cnr] [-T key-table] key command [arguments]
/Users/ladmin/.tmux.conf:56: usage: bind-key [-cnr] [-T key-table] key command [arguments]
```
Relevant config:
```
bind -t vi-copy 'v' begin-selection
bind -t vi-copy 'y' copy-selection
bind -t vi-copy 'r' rectangle-toggle
bind -t vi-copy Escape cancel
```
Permalink to buggy config [here](https://github.com/burtonwilliamt/dotfiles/blob/5772d98d1907cc4c0fd4ee0982e80d804d27123d/.tmux.conf#L53)

**Comment by burtonwilliamt** [comment]
I actually never really use this, so I think I'm going to remove it

**Comment by burtonwilliamt** [comment]
Closed with 95b77a96f5f2a15df76717345413dd2539605ab6
